Classes/Issues/IssuesViewController.swift

```diff
@@ -14,21 +14,24 @@ import SnapKit
 import FlatCache
 import MessageViewController
 
-final class IssuesViewController: MessageViewController,
+final class IssuesViewController:
+    MessageViewController,
     ListAdapterDataSource,
     FeedDelegate,
     AddCommentListener,
     FeedSelectionProviding,
     IssueNeckLoadSectionControllerDelegate,
     FlatCacheListener,
-IssueManagingNavSectionControllerDelegate {
+    IssueManagingNavSectionControllerDelegate,
+    IssueCommentSectionControllerDelegate {
 
     private let client: GithubClient
     private let model: IssueDetailsModel
     private let addCommentClient: AddCommentClient
     private let textActionsController = TextActionsController()
     private var bookmarkNavController: BookmarkNavigationController? = nil
     private var autocompleteController: AutocompleteController!
+    private var focusedCommentModel: IssueCommentModel?
```

Comment:
What's this for? Can we remove?

Reply:
You can

```
> first line
>
> second line
```

Right?

Reply:
That's possible. So you think is a better idea to show the full comment? 
I prefer showing the preview, the comment you replaying is on the same view, easily reachable and the user I am replying knows about the context. 

Otherwise too many information on the new comment cell..

Reply:
What I prefer even more, is when you press on the preview, the view is focused on that comment.